spec/models/profile_photo_spec.rb

```diff
@@ -73,6 +73,14 @@
       expect(subject).to be_valid
     end
 
+    it 'validates image dimensions to be not more than 8192x8192 pixels' do
```

Comment:
I'd prefer that this was broken out into multiple specs–one to test validity and one to test invalidity.